tests/gold_tests/continuations/double.test.py

```diff
@@ -32,7 +32,7 @@
 server = Test.MakeOriginServer("server")
 
 Test.testName = ""
-request_header = {"headers": "GET / HTTP/1.1\r\n\r\n", "timestamp": "1469733493.993", "body": ""}
+request_header = {"headers": "GET / HTTP/1.1\r\nHost: double.test\r\n\r\n", "timestamp": "1469733493.993", "body": ""}
```

Comment:
This request header is [rejected](https://github.com/apache/trafficserver/blob/8d01164257417434b667021365b5f07d0ec596e0/tests/tools/sessionvalidation/sessionvalidation.py#L200-L203) by the microserver, resulting in other responses being discarded. So I had to fix this so that the health check could function.